subsys/mgmt/mcumgr/lib/cmd/img_mgmt/src/zephyr_img_mgmt.c

```diff
@@ -613,7 +615,7 @@ img_mgmt_impl_upload_inspect(const struct img_mgmt_upload_req *req,
 			}
 		}
 
-#if CONFIG_IMG_ERASE_PROGRESSIVELY
+#ifdef CONFIG_IMG_ERASE_PROGRESSIVELY
 		(void) empty;
```

Comment:
Would it be possible to just ifdef definition of the `empty`?

Reply:
I'm not sure I understand 100%, do you mean to omit this change?

It made more sense to have `#ifdef <-> #ifndef` than `#if <-> #ifndef`.

Reply:
```
#ifdef CONFIG_IMG_ERASE_PROGRESSIVELY
	bool empty;
#endif
```

Reply:
It makes no sense to define variable just to basically do equivalent of `ARG_UNUSED` on it.

Reply:
Fixed